zipline/assets/assets.py

```diff
@@ -283,21 +396,21 @@ def lookup_symbol(self, symbol, as_of_date, fuzzy=False):
                 if len(fuzzy_sids) == 1:
                     return self._retrieve_equity(fuzzy_sids[0]['sid'])
 
-            sids = sa.select((equities_cols.sid,)).where(
-                (equities_cols.company_symbol == company_symbol) &
-                (equities_cols.share_class_symbol == share_class_symbol)
-            ).execute().fetchall()
+            sids = list(imap(
+                itemgetter('sid'),
```

Comment:
This is in `lookup_symbol`.   We're only resolving sids for assets that share a symbol.  I think the most-shared symbol in our upstream database has something like 12 assets with that symbol?